BACnetServerExample/BACnetServerExample.cpp

```diff
@@ -610,6 +657,13 @@ int main(int argc, char** argv)
 		// Update values in the example database
 		g_database.Loop();
 
+		// Check for debug messages
+		if (debugMessage.c_str() != g_database.debugMessage && g_database.debugMessage != "") {
```

Comment:
Remove this section, and just do this work in the callback. 

Also just dump the message. No need to prefix it with anything.